spec/lib/topic_query_spec.rb

```diff
@@ -795,8 +796,11 @@
         # includes the invisible topic if you're a moderator
         expect(TopicQuery.new(moderator).list_latest.topics.include?(invisible_topic)).to eq(true)
 
-        # includes the invisible topic if you're an admin" do
+        # includes the invisible topic if you're an admin
         expect(TopicQuery.new(admin).list_latest.topics.include?(invisible_topic)).to eq(true)
+
+        # includes the invisible topic if you're a TL4 user
+        expect(TopicQuery.new(tl4_user).list_latest.topics.include?(invisible_topic)).to eq(true)
```

Comment:
Just wondering, do we have a test case that ensures that `invisible_topic` is not included when viewed by a < TL4 user?

Reply:
Not in `topic_query` test, but would be trivial to add. 🙂

Reply:
I think the "negative" test case is important so that we at least have an assertion failure should the conditional be changed to something incorrect where it always return `true` for all user.

Reply:
Oh, actually it's covered on L:793.

Reply:
Nice :+1: